misc/python/requirements.txt

```diff
@@ -1,5 +1,6 @@
 click==7.1.1
 pg8000==1.15.2
+psutil==5.8.0
```

Comment:
Does `os.cpu_count` not work? We're allergic to additional dependencies in this file because they get installed by users running demos via mzcompose. If not, I think this can be in `requirements-dev.txt` and `mzbench` can request dev dependencies, since it's not something intended for end users.

Reply:
`os.cpu_count` includes hyperthreads, whereas `psutil` can distinguish between logical and physical cores.

I'm happy to add this to a `requirements-dev.txt` and have `mzbench` use that instead.